toolchain/check/type_structure.cpp

```diff
@@ -404,8 +428,9 @@ class TypeStructureBuilder {
 
   Context* context_;
   llvm::SmallVector<WorkItem> work_list_;
-  llvm::SmallVector<int> symbolic_type_indices_;
   llvm::SmallVector<TypeStructure::Structural> structure_;
```

Comment:
I was trying to figure out the semantics of these, maybe:

```suggestion

  // In-progress state for the equivalent `TypeStructure` fields.
  llvm::SmallVector<TypeStructure::Structural> structure_;
```

Reply:
Good comment, thanks